extensions/amp-audio/1.0/test/validator-amp-audio.html

```diff
@@ -0,0 +1,57 @@
+<!--
+  Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  Tests the example from the markdown file, autoplay enabled.
+-->
+<!doctype html>
+<html ⚡ lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="./regular-html-version.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-audio" src="https://cdn.ampproject.org/v0/amp-audio-1.0.js"></script>
+</head>
+<body>
+  <amp-audio width="400" height="300" autoplay src="https://yourhost.com/audios/myaudio.mp3" preload="auto">
+    <div fallback>
+      <p>Your browser doesn’t support HTML5 audio</p>
+    </div>
+    <source type="audio/mpeg" src="foo.mp3">
+    <source type="audio/ogg" src="foo.ogg">
+  </amp-audio>
+
+  <!-- test controlslist attr, which is often provided in camelCase -->
+  <amp-audio width="400" height="300"
+             controlsList="nofullscreen nodownload noremoteplayback foobar">
+  </amp-audio>
```

Comment:
nit: This quirky formatting is another inherited property, we could format this more nicely:

1. Temporarily, add `<!-- prettier-ignore -->` right above the opening `<head>` tag. This will prevent the boilerplate code from formatting.
2. Format with `prettier` locally, or on [prettier.io/playground/](https://prettier.io/playground/) (choose `html` parser on the options on the left side).
3. Remove `<!-- prettier-ignore -->`